modules/omsBidAdapter.js

```diff
@@ -45,13 +45,15 @@ function buildRequests(bidReqs, bidderRequest) {
       const minSize = _getMinSize(processedSizes);
       const viewabilityAmount = _isViewabilityMeasurable(element) ? _getViewability(element, getWindowTop(), minSize) : 'na';
       const viewabilityAmountRounded = isNaN(viewabilityAmount) ? viewabilityAmount : Math.round(viewabilityAmount);
+      const gpidData = _extractGpidData(bid);
 
       const imp = {
         id: bid.bidId,
         banner: {
           format: processedSizes,
           ext: {
-            viewability: viewabilityAmountRounded
+            viewability: viewabilityAmountRounded,
+            ...gpidData
```

Comment:
why put this on imp.banner.ext when it belongs where you pulled it from, on imp.ext?

Reply:
@patmmccann good catch, thank you, fixed